validator/client/validator_test.go

```diff
@@ -142,21 +150,23 @@ func TestWaitActivation_ContextCanceled(t *testing.T) {
 	client := internal.NewMockValidatorServiceClient(ctrl)
 
 	v := validator{
-		key:             validatorKey,
+		keys:            keyMap,
 		validatorClient: client,
 	}
 	clientStream := internal.NewMockValidatorService_WaitForActivationClient(ctrl)
+
 	client.EXPECT().WaitForActivation(
 		gomock.Any(),
 		&pb.ValidatorActivationRequest{
-			Pubkey: v.key.PublicKey.Marshal(),
+			PublicKeys: publicKeys(v.keys),
 		},
 	).Return(clientStream, nil)
 	clientStream.EXPECT().Recv().Return(
 		&pb.ValidatorActivationResponse{
-			Validator: &pbp2p.Validator{
-				ActivationEpoch: params.BeaconConfig().GenesisEpoch,
-			},
+			ActivatedPublicKeys: publicKeys(v.keys),
+			// Validator: &pbp2p.Validator{
```

Comment:
Why comment this out?

Reply:
I wasn't sure it will work with only a list of public keys so i commented it out . forgot to remove it after it all worked out.

Thanks